Layman-eUTxO-Design-Course/batch-1/stateanomaly/week1-homework.md

```diff
@@ -0,0 +1,10 @@
+# DeCo Laymans - Week 1 Homework
```

Comment:
Can you add this into batch-1/individual/stateanomaly/week1-homework.md?

Reply:
Done!